notebooks/processing/PPE_needes_for_covid19_patient_folow.ipynb

```diff
@@ -0,0 +1,424 @@
7zD1QPHAM6rtSbKghcYvfKf/wsXEo5VjTt27HDnaAuuQa7NkqIYU1fOgXE1pRzGtleLGqlYZqkrXWZ1L5ce1bnqXmcVzndoRCI9z0oXfw8YwpM5h4wyBD9wBOOLF4tOB3HEUC46yeojrgkpIBAQqI1A4pOcWw7p5TUnCx0L76Y/WNZDwzA9NLRddqvBg84aOqgHbvjaNgjMzs66G0xl/KIUdNXA7t273Y2HCUbQ6bZpxi0tKOM1IZhIhdy09PZl5MC+/U6Pgw5vadXYtJVnvsJDMHOXM3O6yWShInv27HEPzYzdIQUEWgkBQkhgUDmrBqjObHWewRu1QVdbqZWCLO2CAC8M8nY1xrkZNmPo7ZQrBnbIrl27nDGpXeoR5AwIbGYEmKdhOOb+Nzele0z0LGwU3oKVxsjNPRqjM89NOGGQnbEGGxfnF+0COuSwdoAIGYR6KlVyugqmrJEGOhecAdDYJH05zVVbfQXMWzGgkzhSNmVQJr9txYG7oMX+gBEOImTGYQRGrGxC7vU02NMGOAZsnkQ7UAYYMe7TFsiwGRNOD9NDw5i+ZM4o2gB8eM2gnEGPNPwdDhgNOeYcMqxrQTfVIZPOTyttZoHGyhe9WkORaY5FPcSRBk9sCZSxFTDejHoT6nT5EEi0OhBflIHKHCuIyHcGNAYxjquhwWv/FVfoJmHXLN6ULl+1Q8kBgdUjgL5jtD59+rTkjj0jc8WC3Hjjjc6Ix02fCVBIAYFWR4BJ2fj4uJsIT58+LnsHh+QlN9zi3qLazBPUVm+XIF/zEOBhjwd6HlBmn39W4yJn5ZZbqjrP2L1ZH8qah2jg3CwEmGcwPmNEmD15VLZne+Wmm25yhmIesoOuNgv5wHezImDOd0IGz+hLU3O64vem625yYccwVoU+tVlbPtSrnRBgnjYxMSHnzp2T6Rc0SEpvz6LhvNXqwZjBMz/GZwzRGKBx8DLW8JsP9SGldY9jbGbuO0bwktIqc5LRcGP+2OMM5B4NXrgLuIZ7PzxtvOI3z2ut7HhBbnOKICsOIzACC+SmDsx3uK6gL7ZmiIzTiH+kVOVBGTh6aBP4WjtQFnJsxmQY42ii/uiH6SG/wRg9ImVUD1ldJupk6ZjXXXEIdUeYMcOc/V4y6oDJTVRpbm9utQPrdR3osLaTYQrGlM1vytnMGG9GvQl1urwIJDpfitpZbbCyjouoDGjcUFaicZ3dbMjHdlo7hnbIoUOH3NumDAghBQTaCQH0GQMeun9i7IJkdN8X9PnAgQPujQtuQCEFBFoZAcZlJmZsmMfx2fELsmvHsBw5ckR27tzpJlE2trdyPYJsAYHVIICu4zhnDH/m4hndQ6NbDh48KPv27XMPJ0HnV4NmuLaZCKCjo6OjTlefGjsrOwZ3yOHDh53zhQfsoKvNRD/w3owI8MxKzHsMRcdGzkpK5+6M/8zfQySGzdjioU7tiEBeQ47x4gHp2Nh5jYrU2hElMD6b0Znnf+7PfsJ2xv16ZnZC7+fqSNHr2e843cFKhA7ddF7DmJcyGoKs6oRZTsvL5MxF3eutypFyKINnOPg4Xm3gUIhitMwuqPWgzUn50THpMydAtcp1/80ozqcf/5Hsuv3Nbj6PfdEwsmPdzNrwwijGvh5Cs9B33apLJV3xQojxYs+QdOQ0CkBK9yHK9um5qqKVdTVWuWdHlVas0lj9kqnkhZZCn7mPbjWM21AtgsgtjECitdicJ3Q0Pv6A2RBNB9VsZ9bdnJjsBudLC2tFEC0WAfoARjw8/s7TX6ouBea3vXUQmzGcDAi0CALoMBMnJry88Z/O6oQ+2+nGY8Zl9JrxPqSAwGZCgHEbAxxjdYfqf1Z1njHb5iJB5zdTa7d3XRijCW/K+Ez43059KDZd5Rh0tb3bN0i/8Qgw9mOEov90qqEzk67Of6xfhT618W0SSgwIRBHg+QNDMfYh3rQnXH079E0zflMf7GMk7uN8R/6JqbNS1NUDy/d50VUIqS4NQaZz01ha1tGmZsdkeNtBx5Ny4Mex3RIym/yGEUc+548f03BX1VVCjdQrUyzJs/fdI7e96fbFMtoRo0bq7uehzjjpSFGMoVVmzy9drqtcyp0aDs7t86Lh8oQ9YBZMwou0GXUOKi3VK6m5cyL9R1x+a8etiPESgOFbQKBxBBKdL7Ck83IDsY7Gd0uN0NLa+blx2Md4hWNAoF0QQHdJTH2i+mw0d0H4ExBoUQRs/DV9TXUsH5ftfIuKH8QKCKwaAdN55jLunw7jds6Oq2YaMgQEmoSA6WQFfVUjlP22Y5OKDWwDApsSAes3HFNZNUIVwvi/KRs6VKqtEfD76VIspPaqkm+QZuVLsZiTmfkx3TOjos4UXQmjhm1WuJB4uaIr3SW5Ui6WplZvGZ9So/feqjPHzV91TtDuyTDCjghGz9x1l2TLVadVUt1cRLKESzJqmjx6911S/v2qjdL4J/HaCucNA8M4lapIZu6M26uYPVxI7DFU6ezTPV7mhF1dKnp/NDtvldbvaOlSXlLzFzSDhilbsAMb/62AZahjQGC9EVjxFWc6Lh9StLM1SlvvSgR+AYGAQEAgIBAQCAgEBAICAYGAQEAgIBAQCAgEBAICAYGNRsBsY6y4my/NyMzMpHRg6K6UpaSrXMymhlyvfsk75carXh1Ly6gxfGZuTKbnJhdX0mx0XZpVnmGE8+XoA/dLtrj0cne0zJw6DqazyebKDnVIjTz9nHPkGN8oj63427BAD9P5CanMXdSVLjm3n8siHuqIqei+gp2H3iCdV74plsa1lakzkp4fDRgvAhe+BAQaRyB5NPN4WgfG+WJL2oxcL41VAiEFBAICAYGAQEAgIBAQCAgEBAICAYGAQEAgIBAQCAgEBDYTAqwQcEbvdFnm5qcEB0EaBwznywXngMGmlu3ok57OoUtohsXM7LgUijOLRm87vxmO2A/B6MLjT0gHS1sSUrGvR8p9vQlUXcWo74eXpmfdvl7wDGkJAfQQB1dnuiTl/IwSyrqaZ
```

Comment:
Just an FYI, you could do this with markdown by using `![](../../figs/covid19_patient_flow_ppes_needs.pn)` [markdown image link](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#images)

Reply:
Thanks @lossyrob, this is helpful.
🤦‍♂ I realize that after I committed the changes if people run docker/notebook,  the figs folder is not mounted to the docker, which means if people will still not see the png?